lib/tasks/phlex_tasks.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :phlex do
```

Comment:
Should this be a rake task or a generator? I think most Gems use generators, e.g. `bin/rails g phlex:install` but I might have that wrong as I always copy/paste. 🤔

Reply:
I think it depends: 

- [turbo-rails](https://github.com/hotwired/turbo-rails/blob/main/lib/tasks/turbo_tasks.rake) and [stimulus-rails](https://github.com/hotwired/stimulus-rails/blob/main/lib/tasks/stimulus_tasks.rake) use Rake tasks.
- [Devise](https://github.com/heartcombo/devise/blob/main/lib/generators/active_record/devise_generator.rb) use generators.

I think we can use one or the other. If you have a preference for generators, I can change for generators!

Reply:
Let's stick with tasks then.